Userland/Services/SystemServer/Service.h

```diff
@@ -75,6 +76,12 @@ class Service final : public Core::Object {
     int m_priority { 1 };
     // Whether we should re-launch it if it exits.
     bool m_keep_alive { false };
+    // Whether it was stopped by `service XXX stop`
+    bool m_force_stopped { false };
```

Comment:
Perhaps `m_explicitly_stopped`?

And the comment should be <code>Whether it has been explicitly stopped, such as by &#96;service XXX stop&#96;</code>.